src/outputs/restart.cpp

```diff
@@ -236,13 +240,14 @@ void RestartOutput::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm)
   myDSet = H5Dcreate(file, "/Mesh", PREDINT32, localDSpace, H5P_DEFAULT, H5P_DEFAULT,
                      H5P_DEFAULT);
 
-  auto &nx1 = mb.block_size.nx1;
-  auto &nx2 = mb.block_size.nx2;
-  auto &nx3 = mb.block_size.nx3;
-  int bsize[3] = {nx1, nx2, nx3};
+  auto nx1 = out_ib.e - out_ib.s + 1; // SS mb.block_size.nx1;
+  auto nx2 = out_jb.e - out_jb.s + 1; // SS mb.block_size.nx2;
+  auto nx3 = out_kb.e - out_kb.s + 1; // SS mb.block_size.nx3;
+  int bsize[3] = {mb.block_size.nx1, mb.block_size.nx2, mb.block_size.nx3};
```

Comment:
These comments are confusing to me, what does the SS mean, this kind of looks like you are saying nx1 is the same as mb.block_size.nx1. If so why are you using mb.block_size.nx1 to define bsize and not the more concise nx1 nx2 and nx3 variables?

Reply:
Sorry, SS are my initials and I use // SS or //SS where I think I may have screwed up and may need to go back.